src/cards/thermostat-card/utils.ts

```diff
@@ -32,7 +30,7 @@ export const supportsHeatOnly = (entity: ClimateEntity) => {
 export const getTargetTemps = (entity: ClimateEntity): [number | undefined, number | undefined] => {
     const { target_temp_high, target_temp_low, temperature } = entity.attributes;
 
-    if (supportTargetTemperatureRange(entity)) return [target_temp_low, target_temp_high];
-    if (supportsHeatOnly(entity)) return [target_temp_low ?? temperature, undefined];
-    return [undefined, target_temp_high ?? temperature];
+    if (supportsHeatOnly(entity) || entity.state == "heat") return [target_temp_low ?? temperature, undefined];
+    if (supportsCoolOnly(entity) || entity.state == "cool") [undefined, target_temp_high ?? temperature];
```

Comment:
Would you mind changing these to triple equals (===)?  While not necessary, follows the coding pattern.

Reply:
Good call, I'm not sure how I missed that but I made the switch to strict equality here.